src/test/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationServiceImplTest.java

```diff
@@ -196,4 +243,23 @@ public void testInvalidDate() throws IOException {
         assertTrue(translation.contains("[GRU]"));
     }
 
+  /** Helper Methods **/
+
+  private void initializeTerm() throws Exception {
+      TermResource mockTerm = new TermResource();
+      mockTerm.setId("term id");
+      mockTerm.setLabel("term label");
+      mockTerm.setDefinition("term definition");
+      when(autocompleteService.lookupById(any())).thenReturn(Collections.singletonList(mockTerm));
```

Comment:
All the Collections singleton forms now (in JDK 17) have forms defined directly on the types, which I think are more convenient. For example, `List.of()` can be used without any arguments for an empty list, or with any number of arguments for a constant list. In this case:
```suggestion
      when(autocompleteService.lookupById(any())).thenReturn(List.of(mockTerm));
```

Reply:
We've been using List.of() for some time ... missed this one!